InvenTree/templates/InvenTree/settings/plugin_details.html

```diff
@@ -0,0 +1,73 @@
+{% load inventree_extras %}
+{% load i18n %}
+
+<tr>
+    <td>
+        {% if plugin.is_active %}
+        <span class='fas fa-check-circle icon-green'></span>
+        {% else %}
+        <span class='fas fa-times-circle icon-red'></span>
+        {% endif %}
+
+        {% if plugin.human_name %}
+        {{ plugin.human_name }}
+        {% elif plugin.name %}
+        {{ plugin.name }}
+        {% endif %}
```

Comment:
human_name already uses the name if no title is not present
```suggestion
        {{ plugin.human_name }}
```